docs/classad-attributes/machine-classad-attributes.rst

```diff
@@ -1090,316 +852,227 @@ Machine ClassAd Attributes
-
-``VMOfflineTime``
+:classad-attribute:`VMOfflineTime`
     The time that the VM universe went offline.
 
-:index:`WindowsBuildNumber<single: WindowsBuildNumber; ClassAd machine attribute>`
-
-``WindowsBuildNumber``
+:classad-attribute:`WindowsBuildNumber`
     An integer, extracted from the platform type, representing a build
     number for a Windows operating system. This attribute only exists on
     Windows machines.
 
-:index:`WindowsMajorVersion<single: WindowsMajorVersion; ClassAd machine attribute>`
-
-``WindowsMajorVersion``
+:classad-attribute:`WindowsMajorVersion`
     An integer, extracted from the platform type, representing a major
     version number (currently 5 or 6) for a Windows operating system.
     This attribute only exists on Windows machines.
 
-:index:`WindowsMinorVersion<single: WindowsMinorVersion; ClassAd machine attribute>`
-
-``WindowsMinorVersion``
+:classad-attribute:`WindowsMinorVersion`
     An integer, extracted from the platform type, representing a minor
     version number (currently 0, 1, or 2) for a Windows operating
     system. This attribute only exists on Windows machines.
 
+
 In addition, there are a few attributes that are automatically inserted
 into the machine ClassAd whenever a resource is in the Claimed state:
 
-:index:`ClientMachine<single: ClientMachine; ClassAd machine attribute (in Claimed State)>`
-
-``ClientMachine``
+:classad-attribute:`ClientMachine`
     The host name of the machine that has claimed this resource
 
-:index:`RemoteAutoregroup<single: RemoteAutoregroup; ClassAd machine attribute (in Claimed State)>`
-
-``RemoteAutoregroup``
+:classad-attribute:`RemoteAutoregroup`
     A boolean attribute which is ``True`` if this resource was claimed
     via negotiation when the configuration variable
-    ``GROUP_AUTOREGROUP`` :index:`GROUP_AUTOREGROUP` is ``True``.
+    ``GROUP_AUTOREGROUP`` :classad-attribute:`GROUP_AUTOREGROUP`.
     It is ``False`` otherwise.
 
-:index:`RemoteGroup<single: RemoteGroup; ClassAd machine attribute (in Claimed State)>`
-
-``RemoteGroup``
+:classad-attribute:`RemoteGroup`
     The accounting group name corresponding to the submitter that
     claimed this resource.
 
-:index:`RemoteNegotiatingGroup<single: RemoteNegotiatingGroup; ClassAd machine attribute (in Claimed State)>`
-
-``RemoteNegotiatingGroup``
+:classad-attribute:`RemoteNegotiatingGroup`
     The accounting group name under which this resource negotiated when
     it was claimed. This attribute will frequently be the same as
     attribute ``RemoteGroup``, but it may differ in cases such as when
-    configuration variable ``GROUP_AUTOREGROUP`` :index:`GROUP_AUTOREGROUP` 
-    is ``True``, in which case it will
-    have the name of the root group, identified as ``<none>``.
-
-:index:`RemoteOwner<single: RemoteOwner; ClassAd machine attribute (in Claimed State)>`
+    configuration variable ``GROUP_AUTOREGROUP`` :classad-attribute:`GROUP_AUTOREGROUP`.
```

Comment:
See previous comment.  The :index:`GROUP_AUTOREGROUP` here doesn't mark the start of a new entry but the regex thought it did..